--- a/ledgerautosync/ledgerwrap.py	2022-06-04 13:14:25.337095358 +0200
+++ b/ledgerautosync/ledgerwrap.py	2022-06-04 13:19:43.253573881 +0200
@@ -70,7 +70,10 @@
     def filter_accounts(self, accts, exclude):
         accts_filtered = [a for a in accts if a != exclude]
         if accts_filtered:
-            return accts_filtered[-1]
+            pattern = re.compile("^expense:")
+            return next(filter(lambda account: pattern.match(account),
+                               accts_filtered),
+                        accts_filtered[-1])
         else:
             return None
 
